README.md

```diff
@@ -79,6 +85,20 @@ To set a specific output format, use the `--muxer` option. For example, to outpu
 wf-recorder --muxer=v4l2 --codec=rawvideo --file=/dev/video2
 ```
 
+Microsoft Teams appears to be a bit picky. Install v4l2loopback.ko like this:
```

Comment:
Seems like a good write-up, but I do not think it really fits into the README, maybe we can add this to the wiki and add a link from the readme to it?

Reply:
Absolutely. Can give it a try later, ran out of steam just now.  :-)